gcloud/bigtable/happybase/batch.py

```diff
@@ -133,3 +191,54 @@ def __exit__(self, exc_type, exc_value, traceback):
         # NOTE: For non-transactional batches, this will even commit mutations
```

Comment:
1. See the latest commit.
1. This code wasn't even in this PR, no fair!
1. The behavior is in place to [match the HappyBase][1] implementation.

[1]: https://github.com/wbolster/happybase/blob/9cbd718c10a3089f234f1eac1236b631e1f8e7cd/happybase/batch.py#L125